docs/appendices.txt

```diff
@@ -5,7 +5,7 @@
 [appendix]
 == API Documentation (JavaDoc)
 
-Refer to the JanusGraph API documentation for a complete documentation of all core APIs exposed by JanusGraph.
+Refer to the JanusGraph https://javadoc.io/doc/org.janusgraph/janusgraph-core[API documentation] for a complete documentation of all core APIs exposed by JanusGraph.
```

Comment:
I think the "does not exist" message is fine, because in that case, javadoc.io provides a superb error message, pointing the user to the latest available version.

The other thing is that any user looking at a previously-released version gets API docs relevant to just that version, rather than the latest, so they're not confused about missing or changed features.